client/signup/steps/domains/index.jsx

```diff
@@ -527,10 +527,6 @@ class DomainsStep extends Component {
 		const includeWordPressDotCom = this.props.includeWordPressDotCom ?? ! this.props.isDomainOnly;
 		const promoTlds = this.props?.queryObject?.tld?.split( ',' ) ?? [];
```

Comment:
@southp was there a reason a query param was used for this. This can also be achieved via domain step prop as shown below. I did not refactor it out thinking there might be an alternative use for this. Hoping to get more context from you regarding this.

https://github.com/Automattic/wp-calypso/blob/7bef67c9d5eb53a26a4afb405ff17f79415517eb/client/signup/config/steps-pure.js#L142-L149

Reply:
The domains step actually came with the functionality of designating TLDs through the query parameter individual to this project. Your intuition behind that is correct: there would be more TLD partnerships. Given how the domain step component is being used in multiple places, we need to keep it for sure :)